### PR TITLE
V4.1: Custom range control

### PR DESCRIPTION
**Pull request by gijsbotje**
This pr adds styling to the range input type conform the BS custom control styling
i would like for people on windows to check this codepen: https://codepen.io/gijsbotje/pen/zddJXL
i've tried my best to get it uniform across browsers but i can't really test windows that good

hope you like it :)

schermafbeelding 2017-08-13 om 03 39 23

### CODE REVIEW

**Comment by wolfy1339** [comment]
Please see my PR #19468, I'm sure you can take the styles for Firefox and Chrome for the equivalent of ::-ms-fill-lower

**Comment by gijsbotje** [comment]
@wolfy1339 i tried to make the element look the same accros browsers, that's why in didn't add styling for the progress like you did
only firefox and ie have a way of showing progress

File: scss/_custom-forms.scss [hunk 1]
```diff
@@ -253,3 +253,105 @@
...
+
+  &:focus {
+    outline: none;
+
+    &::-webkit-slider-thumb, &::-moz-range-thumb, &::-ms-thumb {
+      box-shadow: $custom-control-indicator-focus-box-shadow;
+    }
+  }
+
+  &::-moz-focus-inner, &:-moz-focusring {
```
Comment (XhmikosR) [inline scss/_custom-forms.scss]: I think we have one rule per line so try updating your patch.

File: scss/_custom-forms.scss [hunk 2]
```diff
@@ -253,3 +253,110 @@
...
+  }
+
+  // 	IE styling
+  &::-ms-thumb {
+    width: $custom-control-indicator-size;
+    height: $custom-control-indicator-size;
+    background-color: $custom-control-indicator-checked-bg;
+    border: 0;
+    border-radius: $custom-control-indicator-size;
+    -webkit-appearance: none;
```
Comment (XhmikosR) [inline scss/_custom-forms.scss]: I believe this should be unprefixed and let autoprefixer do its thing. The same goes for other prefixes.

**Comment by gijsbotje** [comment]
@XhmikosR fixed it all :)

**Comment by mdo** [comment]
Conflicts resolved and a few more issues being tackled in #25600. Thanks for this, picking it up there.

**Pull request closed without merging**